unattended_scripts/install_functions/opendistro/kibana.sh

```diff
@@ -7,7 +7,7 @@
 
 installKibana() {
     
-    logger "Installing Open Distro for Kibana."
+    logger "Starting the Kibana installation."
```

Comment:
This should be so? or are we already changing the name to Wazuh Dashboard?

Reply:
This should be so, yes. We are first finishing the unification of the installers and when that is finished we will upgrade the names according to the new version.